stacks/cf.py

```diff
@@ -298,20 +328,27 @@ def get_events(conn, stack_name, follow, lines=None):
     '''
     poll = True
     seen_ids = set()
-    next_token = None
+    describe_stack_events_args = {
+        'StackName': stack_name
+    }
+    next_token = False
     try:
         while poll:
-            events = conn.describe_stack_events(stack_name, next_token)
-            next_token = events.next_token
+            #Only send the NextToken argument if we have a token
```

Comment:
I believe `None` in python means that variable is unset, if I recall correctly, so should be fine.